benches/nexmark/generator.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2022 the DBSP contributors.
+// SPDX-License-Identifier: MIT
+
+//! Generators for the models usd in the Nexmark benchmark suite.
+//!
```

Comment:
```suggestion
//! Generators for the models usd in the Nexmark benchmark suite.
//!
```

Reply:
So we shouldn't add copyright and SPDX to new files? (or let me know if you want a separate PR and I'll do the whole repo). Either way, I'll remove it from files in this PR.